scripts/annotation.js

```diff
@@ -6,77 +6,65 @@ function getUrlByParameter(name){
     var length=name.length;
     return url.slice(index+length+1,indexOfEnd);
 }
-
-function get_annotations(){
```

Comment:
I'm sorry but it makes no sense to move the code outside the function. This will create too many issues.
I see that you did this in the other JS scripts as well. The same applies there